packages/react-component-event-listener/test/EventListener-test.tsx

```diff
@@ -4,35 +4,55 @@ import * as React from 'react'
 import * as simulant from 'simulant'
 
 describe('EventListener', () => {
-  it('handles events on `target`', () => {
-    const onClick = jest.fn()
-    const onKeyDown = jest.fn()
+  describe('listener', () => {
+    it('handles events on `target`', () => {
+      const onClick = jest.fn()
+      const onKeyDown = jest.fn()
 
-    mount(
-      <>
-        <EventListener listener={onClick} targetRef={documentRef} type="click" />
-        <EventListener listener={onKeyDown} targetRef={documentRef} type="keydown" />
-      </>,
-    )
+      mount(
+        <>
+          <EventListener listener={onClick} targetRef={documentRef} type="click" />
+          <EventListener listener={onKeyDown} targetRef={documentRef} type="keydown" />
+        </>,
+      )
 
-    simulant.fire(document, 'click')
-    simulant.fire(document, 'keydown')
+      simulant.fire(document, 'click')
+      simulant.fire(document, 'keydown')
 
-    expect(onClick).toHaveBeenCalledTimes(1)
-    expect(onClick).toHaveBeenCalledWith(expect.objectContaining({ type: 'click' }))
+      expect(onClick).toHaveBeenCalledTimes(1)
+      expect(onClick).toHaveBeenCalledWith(expect.objectContaining({ type: 'click' }))
 
-    expect(onKeyDown).toHaveBeenCalledTimes(1)
-    expect(onKeyDown).toHaveBeenCalledWith(expect.objectContaining({ type: 'keydown' }))
-  })
+      expect(onKeyDown).toHaveBeenCalledTimes(1)
+      expect(onKeyDown).toHaveBeenCalledWith(expect.objectContaining({ type: 'keydown' }))
+    })
 
-  it('unsubscribes correctly', () => {
-    const onClick = jest.fn()
+    it('will not rerender on change', () => {
```

Comment:
I don't understand this test.
I would expect that this test was red before the changes and now it is green and helps us avoid any regression.
But is is green even before the change so what's its purpose?

Reply:
We agreed with @kuzhelov to remove this test as it doesn't save us from regressions